scripts/mbedtls_dev/psa_storage.py

```diff
@@ -101,25 +101,48 @@ class Key:
     LATEST_VERSION = 0
     """The latest version of the storage format."""
 
+    EXTENDABLE_USAGE_FLAGS = {
+        Expr('PSA_KEY_USAGE_SIGN_HASH'): Expr('PSA_KEY_USAGE_SIGN_MESSAGE'),
+        Expr('PSA_KEY_USAGE_VERIFY_HASH'): Expr('PSA_KEY_USAGE_VERIFY_MESSAGE')
+    } #type: Dict[Expr, Expr]
```

Comment:
I'm not sure it makes sense to use `Expr` as a dictionary key, since different expressions with the same meaning have different hashes.